wormchain/app/v2_22_0_upgrade.go

```diff
@@ -0,0 +1,50 @@
+package app
+
+import (
+	store "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	packetforwardtypes "github.com/strangelove-ventures/packet-forward-middleware/v4/router/types"
+	ibccomposabilitytypes "github.com/wormhole-foundation/wormchain/x/ibc-composability-mw/types"
+	ibchookstypes "github.com/wormhole-foundation/wormchain/x/ibc-hooks/types"
+	tokenfactorytypes "github.com/wormhole-foundation/wormchain/x/tokenfactory/types"
+)
+
+var V2_22_0_Upgrade = Upgrade{
+	UpgradeName:          "v2.22.0",
+	CreateUpgradeHandler: CreateV2_22_0_UpgradeHandler,
+	StoreUpgrades: store.StoreUpgrades{
+		Added: []string{
+			tokenfactorytypes.ModuleName,
+			ibchookstypes.StoreKey,
+			packetforwardtypes.StoreKey,
+			ibccomposabilitytypes.StoreKey,
+		},
+	},
+}
+
+func CreateV2_22_0_UpgradeHandler(
+	mm *module.Manager,
+	cfg module.Configurator,
+	app *App,
+) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		logger := ctx.Logger().With("upgrade", "v2.22.0")
```

Comment:
nit - I think this release would be v2.22.1? cc @evan-gray

Reply:
probably `v2.23.0` given the break

Reply:
@misko9 ignore this - let's make this change in a follow up PR